### PR TITLE
Flashlight eyes

### PR DESCRIPTION
**Pull request by imtakingabreakdontatme**
:cl: Kor
add: By combing two flashlights and cable coil, you can create a new eye implant: flashlight eyes. People with flashlights for eyes can not see, but they will provide an enormous amount of light to their friends.
/:cl:

I'll let @dannno explain this wonderful idea

### CODE REVIEW

**Comment by kevinz000** [comment]
can you please just let me code proper jcdenton light up eye implants for science?

**Comment by dannno** [comment]
ok so like

**Comment by dannno** [comment]
it's like a personal sacrifice; you are losing your own vision to become a beacon of hope and altruism for the rest of your comrades

**Comment by ExcessiveUseOfCobblestone** [comment]
"utility support class"

**Comment by bgobandit** [comment]
arrest clown, forcibly flashlight implant

**Comment by anconfuzedrock** [comment]
this is fucking great 11/10 but how enormous is it I gotta see it better be really enormous

**Comment by AdamElTablawy** [comment]
M.AddLuminosity(15)

it's pretty enormous

**Comment by anconfuzedrock** [comment]
make that 20

**Comment by PKPenguin321** [comment]
In this code where does it make you blind

**Comment by dannno** [comment]
The eye overlay needs to be painfully visible

File: code/modules/surgery/organs/augments_eyes.dm [hunk 1]
```diff
@@ -66,6 +68,25 @@
 	dark_view = 8
 	aug_message = "You see prey everywhere you look..."
 
+/obj/item/organ/cyberimp/eyes/flashlight
+	name = "flashlight eyes"
+	desc = "It's two flashlights rigged together with some wire. Why would you put these in someones head?"
```
Comment (swindly) [inline code/modules/surgery/organs/augments_eyes.dm]: someone's

**Comment by SweaterKittens** [comment]
_LOOK! LOOK WITH YOUR SPECIAL EYES!_

**Comment by Armhulen** [comment]
**_MY BRAAAND!_**

**Comment by Shadowlight213** [comment]
@PKPenguin321 it applies the welder tint, but extreme so it's all black.

**Comment by MrPerson** [comment]
Merge it if you want, but I expect this to be built maybe 6 times in the next year

**Comment by jughu** [comment]
Now we only need slings back

**Comment by BeeSting12** [comment]
It was pretty funny to be implanted with these and start babbling like a light cultist

**Comment by dannno** [comment]
use this
http://a.pomf.cat/vnchey.dmi
the eyes have to be like some empurata shit
you gotta see a person with flashlight eyes and instantly know and be like "oh damn"

**Comment by XDTM** [comment]
Finally i can blind my surgery victims

**Comment by Supermichael777** [comment]
@dannno you set of my crappy browser antivirus fix your shit

**Comment by dannno** [comment]
i'm about to set off my fist up your ass if you ever tell me what to do again

**Comment by FantasticFwoosh** [comment]
I can't just see why you couldn't...

> A. justify this to add add a eyeballs body part so people's eyes pop and fly out of their head when they get gibbed etc.

Or 

B. Just have this be a ghetto version of a R&D eye-implant to which you must choose between this or welding implant because there's only space for one

**Comment by XDTM** [comment]
Kor is working on eye organs but vision code is scattered all over the place.
Eye implants are actually non-exclusive in most cases. HUD implants are non-compatible with each other and thermal and x-ray as well, but that's it.

**Comment by ChangelingRain** [comment]
i can't see

**Pull request merged**